### PR TITLE
fix title for municipality

### LINKED ISSUE #408: Format of municipality on first page is not conform to specification
Actual situation is like:
grafik

Specification says:
grafik

### PR DESCRIPTION
**Pull request by vvmruder**
fixes #408

### CODE REVIEW

**Comment by vvmruder** [comment]
I need help. How do we achieve shifting the municipality id_bfs to have it like the specification defines?
see #421

**Comment by sbrunner** [comment]
The issue link is wrong ...

**Comment by vvmruder** [comment]
sorry about the confusion

I changed it to the correct link :)

**Comment by sbrunner** [comment]
Thanks :-)

**Comment by kdeininger** [comment]
image

**Comment by vvmruder** [comment]
Since this is due to our data I close this PR

File: print/print-apps/oereb/Report_de.properties [hunk 1]
```diff
@@ -21,7 +21,7 @@ ReducedPublicLawRestrictionsPDFExtract = Auszug aus dem Kataster der öffentlich
 FullPublicLawRestrictionsPDFExtract = Auszug aus dem Kataster der öffentlich-rechtlichen Eigentumsbeschränkungen (ÖREB-Kataster)
 RealEstateLabel = Grundstück-Nr
 EGRID = E-GRID
-MunicipalityLabel = Gemeinde
+MunicipalityLabel = Gemeinde (BFS-Nr.)
```
Comment (sbrunner) [inline print/print-apps/oereb/Report_de.properties]: Don't we want that?

**Pull request merged**